packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/authentication/index.hbs

```diff
@@ -0,0 +1,53 @@
+<ActionCardContainer
+  @isComplete={{@isComplete}}
+>
+  <Boxel::ActionChin
+    @state={{this.authState}}
+  >
+    <:default as |d|>
+      {{#if (and this.error this.authenticationTask.isIdle)}}
```

Comment:
I think we should handle `@frozen` layer 2 not being connected.

Reply:
I actually removed @frozen because the memorialized state does not have any interactive elements. Do you think we still need it?

Reply:
Yep, we need it for the default state (user performs action that cancels workflow prior to authenticating, eg. disconnecting layer 2).